indexer/postgres/base_sql.go

```diff
@@ -17,17 +17,20 @@ CREATE TABLE IF NOT EXISTS tx
     id             BIGSERIAL PRIMARY KEY,
     block_number   BIGINT NOT NULL REFERENCES block (number),
     index_in_block BIGINT NOT NULL,
-    data           JSONB  NOT NULL
+    data           JSONB NULL,
+    bytes          BYTEA NULL 
 );
 
 CREATE TABLE IF NOT EXISTS event
 (
     id           BIGSERIAL PRIMARY KEY,
     block_number BIGINT NOT NULL REFERENCES block (number),
-    tx_id        BIGINT NULL REFERENCES tx (id),
-    msg_index    BIGINT NULL,
-    event_index  BIGINT NULL,
-    type         TEXT   NOT NULL,
-    data         JSONB  NOT NULL
+    block_stage  INTEGER NOT NULL,
+    tx_index     BIGINT NOT NULL,
+    msg_index    BIGINT NOT NULL,
+    event_index  BIGINT NOT NULL,
+    type         TEXT   NULL,
+    data         JSONB  NULL,
+    attributes   JSONB  NULL
```

Comment:
I don't think we should store both data and attributes. I think if an event has `Data`, we should use that. If it doesn't have data and just has `Attributes` we should store that in `data` as JSON

Reply:
So basically I think we just need one `data` column, otherwise it's a lot of duplicated data in most cases

Reply:
Can we make sure it's documented on the Event struct that Data and Attributes should represent the same thing and that Data is preferred by indexers if it is present.